index.html

```diff
@@ -1,15 +1,138 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
     <meta charset="UTF-8">
-    <title>First</title>
+    <title>Trip house</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+
 </head>
 <body>
-
-<h1>My First Page</h1>
-<a href="./about/index.html">About Author</a>
-
-
+<div class="top-section">
+    <header class="header">
+        <img src="./image/logo_vector.png" alt="logo">
+        <ul class="menu">
+            <li><a class="item-stays" href="#"></a>Stays</li>
+            <li><a class="item-attractions" href="#"></a>Attractions</li>
+            <li><a class="item-night" href="/">
+                <img src="./image/Night.png" alt="Night"></a>
+            </li>
+            <li><a class="menu-item" href="/">
+                <img src="./image/AccountCircle.png" alt="Accountcircle"></a>
+            </li>
+        </ul>
+    </header>
+    <h1>Discover stays<br>
+        to live, work or just relax</h1>
+    <div class="choose">
+        <div class="text-1">Your destination or hotel name</div>
+        <div class="text-2">Check-in — Check-out</div>
```

Comment:
this label jump to the left on my screen and it looks like 2 labels for "destination" input - try to set labels with absolute position and top option